api/voice-react-native-sdk.api.md

```diff
@@ -297,6 +324,36 @@ export namespace CallKit {
     }
 }
 
+// @public
+export class CallMessage {
+    // Warning: (ae-forgotten-export) The symbol "NativeCallMessageInfo" needs to be exported by the entry point index.d.ts
+    //
+    // @internal
+    constructor({ content, contentType, messageType, voiceEventSid, }: NativeCallMessageInfo);
+    _content: string;
+    _contentType: CallMessage.ContentType;
+    getContent(): string;
+    getContentType(): CallMessage.ContentType;
+    getMessageType(): CallMessage.MessageType;
+    getSid(): string | undefined;
+    _messageType: CallMessage.MessageType;
+    _voiceEventSid?: string;

```

Comment:
These members are public, because I have `OutgoingCallMessage` implement `CallMessage`, if I make them private I get type error.

https://github.com/twilio/twilio-voice-react-native/blob/VBLOCKS-2464/feat-call-message/src/OutgoingCallMessage.ts#L154